src/scripts/webview.js

```diff
@@ -41,6 +42,24 @@ class WebView extends EventEmitter {
                 }, '*')
             `);
         });
+
+        Mousetrap.bind(['command+left', 'ctrl+left'], () => {
```

Comment:
Looks like you should use "[mod+left](https://craig.is/killing/mice#api.bind.combo)" instead for cross platform, rather than binding both.